### PR TITLE
C24_WMDE_Desktop_DE_07

### PR DESCRIPTION
**Pull request by Sperling-0**
- Both banners:
        - are based on the control banner of desktop-de-06.

- Variant banner:
       - The variant banner is revived from [desktop-de-02](https://de.wikipedia.org/?banner=B24_WMDE_Desktop_DE_02_var&devMode).
       - The variant banner's headline is changed to "Wikipedia ist unverkäuflich".

- Use swatches in the fallback banner
- Change BIC
- Inline UoF infographics

Ticket: https://phabricator.wikimedia.org/T375603

### CODE REVIEW

File: banners/desktop/C24_WMDE_Desktop_DE_07/styles/colors.ts
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_07/styles/colors.ts:1]: You can delete this

File: banners/desktop/C24_WMDE_Desktop_DE_07/styles/fallback_colors.ts
Comment (Abban) [inline banners/desktop/C24_WMDE_Desktop_DE_07/styles/fallback_colors.ts:1]: And delete this too

File: test/banners/desktop/components/FallbackBanner.spec.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { afterEach, beforeEach, describe, test, vi } from 'vitest';
 import { mount, VueWrapper } from '@vue/test-utils';
-import FallbackBanner from '@banners/desktop/C24_WMDE_Desktop_DE_03/components/FallbackBanner.vue';
+import FallbackBanner from '@banners/desktop/C24_WMDE_Desktop_DE_07/components/FallbackBanner.vue';
```
Comment (Abban) [inline test/banners/desktop/components/FallbackBanner.spec.ts:3]: Oh right, we've been forgetting all about the fallback banner!

**Review by Abban — APPROVED** [review]

**Comment by Sperling-0** [comment]
⚠️ Rebase this PR from latest main.

To get the latest UoF working:

- For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
- For Mobile banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

**Comment by Sperling-0** [comment]
> ⚠️ Rebase this PR from latest main.
> 
> To get the latest UoF working:
> 
>     * For Desktop banners: `BannerCtrl.vue`, `BannerVar.vue`, `FallbackBanner.vue` needs to be updated. (If it is DE desktop include DE infographic and vice versa)
> 
>     * For Mobile banners: `BannerCtrl.vue`, `BannerVar.vue` needs to be updated. (If it is DE mobile include DE infographic and vice versa)

✅ This branch is rebased to latest main. Has 'Inline UoF infographics' changes according to https://github.com/wmde/fundraising-banners/pull/563

**Review by moiikana — APPROVED** [review]

**Pull request merged**